mlir/test/Conversion/SCFToAffine/lower-affine-parallal.mlir

```diff
@@ -0,0 +1,32 @@
+// RUN: imex-opt %s -lower-affine -allow-unregistered-dialect -split-input-file| FileCheck %s
```

Comment:
filename should be "lower-affine-parallel.mlir" ('a'->'e')

Reply:
Done